wdl/ClusterBatch.wdl

```diff
@@ -65,6 +65,9 @@ workflow ClusterBatch {
 
     Float? java_mem_fraction
 
+    # PlotSVCountsPerSample metrics
+    Int N_IQR_cutoff_plotting = 6
```

Comment:
Not required but it could be good to move this parameter higher in the list of inputs, above the "Module metrics parameters" section since it is a main workflow input